docs/HowTo/clusterconnectivity/klaw-core-with-clusterapi.md

```diff
@@ -6,8 +6,8 @@ respective `application.properties` files and verifying the connectivity between
 
 You can find the `application.properties` file located in the following paths:
 
-- Core: [klaw/core/src/main/resources]
-- Cluster-api: [klaw/cluster-api/src/main/resources]
+- Core: [Klaw/core/src/main/resources]
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one? 

In case this pops up as error for the "Klaw Core" rule, you can put in backticks instead of brackets, which is the correct markdown formatting for code.

Instead, the line should be changed to start with "Klaw Core" instead of just "Core".